README.md

```diff
@@ -1,24 +1,50 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![](https://img.shields.io/badge/Microverse-blueviolet) ![](https://img.shields.io/badge/MENGSTU-FENTAW-success)
 
-Things you may want to cover:
+# Blog App - Ruby on Rails
 
-* Ruby version
+The Blog app is a classic example of a blog website that is fully functional website. It includes the list of posts and empower readers to interact with them by adding comments and liking posts. The app is build with consideration of the followinf data model(Entity Relationship Diagram):
+![](blog_app_erd.png)
 
-* System dependencies
 
-* Configuration
 
-* Database creation
+## Built with
 
-* Database initialization
+    - Major languages: Ruby, Ruby on Rails
+    - Technologies used: Rubocop, Gitflow
 
-* How to run the test suite
+## Prerequisites
 
-* Services (job queues, cache servers, search engines, etc.)
+    - Ruby installed on local system
+    - npm ( Node Package Manager )
+    - Git and Github Account
 
-* Deployment instructions
+## Getting Started
 
-* ...
+To get a local copy up and running follow these simple example steps.
+
+- Open terminal
+- Change to the desired directory then run the following:
+
+  - Run `git clone git@github.com:mengiefen/blog-app.git`
+
+  - cd repo-name/ 
```

Comment:
- [ ] (Optional) After changing directory, you can advise those who wish to clone your project to do the following:

```
git switch <branch name>
run `bundle install`
run `rails db:create`
run `rails db:migrate`
```